.gitignore

```diff
@@ -8,3 +8,6 @@ __pycache__/
 /.venvs/
 /docs/assets
 /docs/build/
+.python-version
+.idea/
```

Comment:
Pro tip: you can have [a global gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file) for IDE-specific files so you don't have to update .gitignore for every project you contribute to. I don't mind the change, though, I just thought you might find it useful :)

Reply:
Oh that is handy, thanks!